src/aya/util/FileUtils.java

```diff
@@ -1,53 +1,30 @@
 package aya.util;
 
-import java.io.BufferedReader;
+import aya.AyaPrefs;
+
 import java.io.File;
-import java.io.FileInputStream;
 import java.io.IOException;
-import java.io.InputStreamReader;
 import java.nio.charset.StandardCharsets;
-
-import aya.AyaPrefs;
+import java.nio.file.Files;
 
 public class FileUtils {
-	
+
 	@SuppressWarnings("null")
-	public static String readAllText(String path) throws IOException {
-		File file = new File(path);
-		BufferedReader br = null;
-		StringBuilder sb = new StringBuilder();
-		
-		try {
-			String line;
-			br = new BufferedReader(new InputStreamReader(new FileInputStream(file), StandardCharsets.UTF_8));
-			while((line = br.readLine()) != null) {
-				sb.append(line+'\n');
-			}
-		} catch (IOException e) {
-			try {
-				br.close();
-			} catch (NullPointerException e2) {}
-			throw e;
-		}
-		
-		br.close();
-		return sb.toString();
+	public static String readAllText(File file) throws IOException {
+		return new String(Files.readAllBytes(file.toPath()), StandardCharsets.UTF_8); // in Java 11 you can also do Files.readString(Path)

```

Comment:
Note that this is not exactly equivalent to the previous implementation, as `\r\n` is not trimmed to `\n`.
I think this is preferable when working with binary files. (Unless there's an operator for reading binary files that I missed?)